src/sdk/bucketspace_fs.js

```diff
@@ -318,8 +319,8 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
             name,
             tag: js_utils.default_value(tag, undefined),
             owner_account: account._id,
-            system_owner: new SensitiveString(account.email),
-            bucket_owner: new SensitiveString(account.email),
+            system_owner: new SensitiveString(account.name),
+            bucket_owner: new SensitiveString(account.name),
```

Comment:
From your answer (see [comment](https://github.com/noobaa/noobaa-core/issues/7792#issuecomment-1934444768)) I understand that in `bucketspace_fs` is a part of the whole noobaa endpoint process in which we have the account email, so if you can explain this change.

Reply:
The comment is about why keep sensitive string on bucketspaceFS - it's indeed a part of the endpoint and that's why it's a sensitive string.
I suppose you meant to ask about our discussion from yesterday (https://github.com/noobaa/noobaa-core/pull/7801#discussion_r1485549529).
Principles of bucket policies on NC are names and not emails as opposed to containerized NooBaa in which the bucket policy principles are emails. But after the merge of https://github.com/noobaa/noobaa-core/pull/7801 it doesn't matter because email is actually the name